src/mpyl/steps/deploy/bpm_deploy.py

```diff
@@ -0,0 +1,61 @@
+"""Deploys the Camunda diagrams in the build stage to Camunda cluster, using BPM. """
+
+from logging import Logger
+from typing import List
+from functools import reduce
+from .bpm import deploy_to_cluster, deploy_to_modeler
+from ...utilities.bpm import CamundaConfig
+from . import STAGE_NAME
+from .. import Step, Meta
+from ..models import Input, Output, ArtifactType
+
+
+class BpmDiagramDeploy(Step):
+    def __init__(self, logger: Logger) -> None:
+        super().__init__(
+            logger,
+            Meta(
+                name="BPM Diagram Deploy",
+                description="Deploy BPM diagram to Camunda Cluster",
+                version="0.0.1",
+                stage=STAGE_NAME,
+            ),
+            produced_artifact=ArtifactType.NONE,
+            required_artifact=ArtifactType.NONE,
+            # TO DO: make an artifcatype with a camunda deployed diagram link
```

Comment:
Do you still want to do this? If not within this PR maybe you could create a follow-up card and link it here in the comment?